tools/ci_build/github/linux/docker/inference/x86_64/python/openvino/Dockerfile

```diff
@@ -0,0 +1,68 @@
+# Use the specified UBI8 base image with GCC 14
+ARG BASEIMAGE="onnxruntimebuildcache.azurecr.io/internal/azureml/onnxruntime/build/cpu_x64_ubi8_gcc14:20250124.1"
+FROM ${BASEIMAGE}
+
+ARG BUILD_UID=1000
+ARG BUILD_USER=onnxruntimedev
+
+# Switch to root for system-level installations
+USER root
+WORKDIR /
+
+RUN dnf install -y --nodocs \
+        wget \
+        tar \
+        gzip \
+        git \
+        numactl-libs \
+        ocl-icd \
+    && dnf clean all \
+    && rm -rf /var/cache/dnf
+
+ENV INTEL_OPENVINO_DIR=/opt/intel/openvino_2025.0.0
+ARG OPENVINO_PACKAGE_URL=https://storage.openvinotoolkit.org/repositories/openvino/packages/2025.0/linux/openvino_toolkit_rhel8_2025.0.0.17942.1f68be9f594_x86_64.tgz
+ARG TEMP_DIR=/tmp/openvino_installer
+
+RUN mkdir -p ${TEMP_DIR} && \
+    cd ${TEMP_DIR} && \
+    wget -O openvino_toolkit.tgz ${OPENVINO_PACKAGE_URL} && \
+    tar -xzf openvino_toolkit.tgz && \
+    OV_FOLDER_NAME=$(ls -d openvino_toolkit_rhel8_*) && \
+    mkdir -p /opt/intel && \
+    mv ${OV_FOLDER_NAME} ${INTEL_OPENVINO_DIR} && \
+    ln -s ${INTEL_OPENVINO_DIR} /opt/intel/openvino && \
+    # Clean up installer files
+    cd / && rm -rf ${TEMP_DIR}
+
+RUN dnf install -y --nodocs \
+        libedit \
+        ocl-icd \
+        https://repositories.intel.com/graphics/rhel/8.6/intel-gmmlib-22.3.1-i529.el8.x86_64.rpm \
+        https://repositories.intel.com/graphics/rhel/8.6/intel-igc-core-1.0.12504.6-i537.el8.x86_64.rpm \
+        https://repositories.intel.com/graphics/rhel/8.6/intel-igc-opencl-1.0.12504.6-i537.el8.x86_64.rpm \
+        https://repositories.intel.com/graphics/rhel/8.6/intel-opencl-22.43.24595.35-i538.el8.x86_64.rpm \
+        https://repositories.intel.com/graphics/rhel/8.6/intel-level-zero-gpu-1.3.24595.35-i538.el8.x86_64.rpm \
+        https://repositories.intel.com/graphics/rhel/8.6/level-zero-1.8.8-i524.el8.x86_64.rpm \
+    && dnf clean all \
+    && rm -rf /var/cache/dnf
+
+# --- Environment Setup ---
```

Comment:
Can't we use `source <openvino_install_directory>/setupvars.sh` as mentioned in the https://onnxruntime.ai/docs/execution-providers/OpenVINO-ExecutionProvider.html#usage?

Reply:
It only impacts the current shell. If we put it there, once the image is built, the effect is gone.